common/scala/src/main/scala/whisk/core/entity/Exec.scala

```diff
@@ -46,37 +49,55 @@ import whisk.core.entity.ArgNormalizer.trim
 sealed abstract class Exec(val kind: String) {
```

Comment:
I think I mentioned this before... I would have added a trait `ByteSizeable` with a `size` method in the trait to mix in with whatever we'd want to size (including `LogLimit`).